### ISSUE TITLE
GraphFrames example notebook fails

### ISSUE THREAD

**Issue by markwatsonatx** [ibm-watson-data-lab/pixiedust#294]
As a user, I want to import the GraphFrames package

### Expected behavior

Able to run the GraphFrames example notebook

### Actual behavior

Notebook fails when you try to use display API with the following error:

`java.lang.NoClassDefFoundError: com/typesafe/scalalogging/slf4j/LazyLogging`

Fails locally and on DSX

### Steps to reproduce the behavior

Run the GraphFrames example notebook

**Comment by vabarbosa** [comment]
recent graphframes version now have dependencies on additional libraries. if you are using an older graphframes package (e.g., 0.1) it should work. however, if you install a new version then you will need to install the additional libraries.

```
pixiedust.installPackage("com.typesafe.scala-logging:scala-logging-api_2.11:2.1.2")
pixiedust.installPackage("com.typesafe.scala-logging:scala-logging-slf4j_2.11:2.1.2")
```
the notebook has been updated with this information in commit https://github.com/ibm-cds-labs/pixiedust/pull/370/commits/c1600313cffa4ef4045a7730301ec14929297b97

**Comment by vabarbosa** [comment]
addressed in pr #370 which will be included in v1.0.7
